### DISCUSSION TITLE
[Request] Cineform support

### DISCUSSION THREAD

**Discussion by BuyMyMojo** [Xaymar/obs-stream-effects#1005] [Ideas & Feedback]
I'd like to request Cineform encoder support. When recording I prefer to have Cineform as my intermediate format as it is easier to encode than ProRes on my window machine and still supported by MPV for playback.

This is a lower-priority request but it'd still be a nice to have, thank you!

Relevant link: https://github.com/gopro/cineform-sdk

**Comment by Xaymar** [comment]
For further feature requests, please join [the official Discord](https://s.xaymar.com/streamfx-dc) and create a thread under #development. Thank you.
